packages/bierzo-wallet/src/routes/account/register/components/SelectAddressesTable.tsx

```diff
@@ -92,6 +92,8 @@ const AddressRow = ({
     }
   };
 
+  const isStarname = addressItem.chain.chainName.toLocaleLowerCase().indexOf("starname") !== -1;
```

Comment:
@antoineherzog, I know it's not your fault, but having two versions of SelectAddressesTable.tsx is awful, ie not maintainable.  The `src/components/AccountEdit/SelectAddressesTable.tsx` should be the single canonical version.

Reply:
I agree, however I would like to keep it like that for now for this PR. I spent lots of time of testing and some test are not passing. I would like to finish it

Reply:
OK, we can deal with the multiple `SelectAddressesTable.tsx`s in another PR.